diff --git a/dist/index.d.cts b/dist/index.d.cts
index 63c105ddb73f8496818bae835843778cf4fc5b28..bf200a8b1c385748eb7885817d4f6642d37ed522 100644
--- a/dist/index.d.cts
+++ b/dist/index.d.cts
@@ -1,6 +1,6 @@
 import { Command } from 'prosemirror-state';
 
-declare module 'prosemirror-model' {
+// declare module 'prosemirror-model' {
     interface AttributeSpec {
         /**
          * Indicates if the block can be split using the `splitBlockAs` command.
@@ -15,7 +15,7 @@ declare module 'prosemirror-model' {
          */
         splittable?: boolean;
     }
-}
+// }
 /**
  * Split the block at the current selection, but try to inherit splittable
  * attributes from the previous block.
diff --git a/dist/index.d.ts b/dist/index.d.ts
index 63c105ddb73f8496818bae835843778cf4fc5b28..bf200a8b1c385748eb7885817d4f6642d37ed522 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,6 +1,6 @@
 import { Command } from 'prosemirror-state';
 
-declare module 'prosemirror-model' {
+// declare module 'prosemirror-model' {
     interface AttributeSpec {
         /**
          * Indicates if the block can be split using the `splitBlockAs` command.
@@ -15,7 +15,7 @@ declare module 'prosemirror-model' {
          */
         splittable?: boolean;
     }
-}
+// }
 /**
  * Split the block at the current selection, but try to inherit splittable
  * attributes from the previous block.
