diff --git a/dist/index.d.ts b/dist/index.d.ts
index fef3a58fabdee10f5c7355bec86767995be60414..e662aa69070358ca97696c7184ce1457809219f5 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,7 +1,6 @@
 import { Node, ResolvedPos, NodeType, Attrs } from 'prosemirror-model';
 import { Command } from 'prosemirror-state';
 
-declare module 'prosemirror-model' {
     interface AttributeSpec {
         /**
          * Indicates if the block can be split using the `splitBlockAs` command.
@@ -16,7 +15,6 @@ declare module 'prosemirror-model' {
          */
         splittable?: boolean;
     }
-}
 /**
  * Split the block at the current selection, but try to inherit splittable
  * attributes from the previous block.
