diff --git a/dist/index.d.ts b/dist/index.d.ts
index c027d663a5072652d29a100003a8454c4fd3208f..429a6900f868c694eeb4703f592bad86b930449d 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -900,7 +900,7 @@ A description of a node type, used when defining a schema.
 interface NodeSpec {
     /**
     The content expression for this node, as described in the [schema
-    guide](/docs/guide/#schema.content_expressions). When not given,
+    guide](https://prosemirror.net/docs/guide/#schema.content_expressions). When not given,
     the node does not allow any content.
     */
     content?: string;
@@ -1419,7 +1419,7 @@ structure between the old and new data as much as possible, which a
 tree shape like this (without back pointers) makes easy.
 
 **Do not** directly mutate the properties of a `Node` object. See
-[the guide](/docs/guide/#doc) for more information.
+[the guide](https://prosemirror.net/docs/guide/#doc) for more information.
 */
 declare class Node {
     /**
@@ -1451,7 +1451,7 @@ declare class Node {
     readonly text: string | undefined;
     /**
     The size of this node, as defined by the integer-based [indexing
-    scheme](/docs/guide/#doc.indexing). For text nodes, this is the
+    scheme](https://prosemirror.net/docs/guide/#doc.indexing). For text nodes, this is the
     amount of characters. For other leaf nodes, it is one. For
     non-leaf nodes, it is the size of the content plus two (the
     start and end token).
