diff --git a/src/internal/client/index.js b/src/internal/client/index.js
index 5a4c8b49bb763552779f3b652e1df6151c4aed1b..e742ccf197dedc8f65ee7b77a1b00be8727712e9 100644
--- a/src/internal/client/index.js
+++ b/src/internal/client/index.js
@@ -158,6 +158,7 @@ export {
 } from './dom/operations.js';
 export { noop } from '../shared/utils.js';
 export {
+	add_snippet_symbol,
 	validate_component,
 	validate_dynamic_element_tag,
 	validate_snippet,
